core/src/test/java/bio/terra/pearl/core/service/kit/LivePepperDSMClientIntegrationTest.java

```diff
@@ -2,26 +2,68 @@
 
 import bio.terra.pearl.core.BaseSpringBootTest;
 import bio.terra.pearl.core.IntegrationTest;
+import bio.terra.pearl.core.dao.kit.KitTypeDao;
 import bio.terra.pearl.core.factory.kit.KitRequestFactory;
 import bio.terra.pearl.core.factory.participant.EnrolleeFactory;
-import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.function.Executable;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.transaction.annotation.Transactional;
 
+import java.util.UUID;
+
+import static com.github.seregamorph.hamcrest.MoreMatchers.where;
 import static org.hamcrest.MatcherAssert.assertThat;
-import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.*;
+import static org.junit.jupiter.api.Assertions.assertThrows;
 
 public class LivePepperDSMClientIntegrationTest extends BaseSpringBootTest {
 
+    public static final Logger log = LoggerFactory.getLogger(LivePepperDSMClientIntegrationTest.class);
+
+    public static final String STUDY_SHORTCODE = "ourheart";
+
     @Autowired
     private LivePepperDSMClient livePepperDSMClient;
 
+    @Disabled("Avoid creating endless test data in dev DSM")
```

Comment:
This can be enabled and run sparingly to test successful creation of kits in DSM.

Reply:
Nice.  what do you recommend is the frequency we run this?  Maybe every two weeks to coincide with Pepper releases?  I'm wondering if we should put running this test on our monitoring checklist--otherwise I'm certain we will forget

Reply:
I like the idea of having it on our monitoring checklist, which can also mention to pay attention to Pepper release timing for additional opportunities to run it. I'll add that.